### PR TITLE
Webgui new default ui

### PR DESCRIPTION
**Pull request by supertick**

### CODE REVIEW

File: src/main/java/org/myrobotlab/service/Agent.java [hunk 1]
```diff
@@ -1047,10 +1047,10 @@ public Process spawn(CmdOptions inOptions) throws IOException, URISyntaxExceptio
     }
 
     if (options.services.size() == 0) {
-      options.services.add("log");
-      options.services.add("Log");
-      options.services.add("gui");
-      options.services.add("SwingGui");
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/Agent.java]: does this mean swing gui doesn't start by default anymore?

**Comment by kwatters** [comment]
i'm all for webgui starting by default.. however i am against disabling the swing gui starting by default.

**Comment by kwatters** [comment]
additionally.. if this pull request is actually just to change the default gui.. it should only contain the 1 file.. not a whole bunch of stuff related to webkit speech synth.

**Comment by supertick** [comment]
gah .. the swing gui lives on .... it just won't die !

**Pull request merged**